src/components/nav/styled-nav-top.tsx

```diff
@@ -32,7 +32,7 @@ export default function StyledNavTop() {
     },
     {
       name: "TFT Bulk Tool",
-      href: `/tft/bulk-tool?league=Crucible`,
+      href: `/tft/bulk-tool?league=${league}`,
```

Comment:
The bulk tool is only allowed to be Crucible or Standard

Reply:
I see, my mistake